backend/src/modules/users/dto/create-users.dto.ts

```diff
@@ -23,7 +23,7 @@ export class CreateUserDto {
   @IsString()
   password!: string;
 
-  @IsUUID()
+  @IsString()
```

Comment:
@IsEnum check if provided value equals to student, teacher, try with upper case and lower case, mb use class transformer